clang/lib/Lex/HeaderSearch.cpp

```diff
@@ -1773,8 +1773,11 @@ HeaderSearch::lookupModuleMapFile(const DirectoryEntry *Dir, bool IsFramework) {
 
 Module *HeaderSearch::loadFrameworkModule(StringRef Name, DirectoryEntryRef Dir,
                                           bool IsSystem) {
-  if (Module *Module = ModMap.findModule(Name))
-    return Module;
+  // CAS: we need to load modulemaps even if we have the primary module for this
```

Comment:
@jansvoboda11 do you have any concerns with this change?

Reply:
To clarify, when we upstream CAS/caching, my intention would be to just delete this check.  I'm only keeping this comment for the benefit of merge conflict handling.

Reply:
I found a case where this causes "error: redefinition of module". Looking into what else can work here.

Reply:
This code is removed. I am instead triggering module lookups during scanning a PCH that should behave the same as the module lookups we would do when reading modules without include-tree, and cause us to lookup the modulemap files we need.